drivers/download_stage1_psuedo.R

```diff
@@ -15,27 +17,40 @@ dates_pseudo <- seq(as.Date("2020-09-24"), Sys.Date(), by=1)
 
 message("GEFS v12 stage1-stats")
 bench::bench_time({ # thelio
-  s3 <- gefs_s3_dir("stage1-stats")
+  s3 <- gefs4cast::gefs_s3_dir(product = "stage1-stats",
+                               path = "", # should this path be more specific? the noaa bucket in the config is "drivers/noaa/gefs-v12-reprocess/"
+                               endpoint = config$endpoint,
+                               bucket = config$driver_bucket)
   have_dates <- gsub("reference_datetime=", "", s3$ls())
   missing_dates <- dates[!(as.character(dates) %in% have_dates)]
-  gefs_to_parquet(missing_dates,
-                  ensemble=c("geavg", "gespr"),
-                  path = s3,
-                  sites = sites)
+  gefs4cast::gefs_to_parquet(dates = missing_dates,
+                             ensemble = c("geavg", "gespr"),
+                             path = s3,
+                             sites = sites) # should partitioning also include the project_id ??
```

Comment:
somewhat similar to the question above on how we want to organize the driver files - should there be partitioning on the `project_id` in addition to `reference_datetime` and `site_id`?

Reply:
Lets not partition by project_id but have project_id in the path. see above